index.js

```diff
@@ -180,72 +180,91 @@
   //  Value :: Location
   var Value = 'Value';
 
-  //  _funcPath :: (Boolean, Array String, a) -> Nullable Function
-  function _funcPath(allowInheritedProps, path, _x) {
-    var x = _x;
-    for (var idx = 0; idx < path.length; idx += 1) {
-      var k = path[idx];
-      if (x == null || !(allowInheritedProps || has(k, x))) return null;
-      x = x[k];
-    }
-    return typeof x === 'function' ? x : null;
+  //  prefix :: String
+  var prefix = 'fantasy-land/';
+
+  //  funcProp :: (String, a) -> Nullable Function
+  function funcProp(prop, x) {
+    return x != null && typeof x[prop] === 'function' ? x[prop] : null;
   }
 
-  //  funcPath :: (Array String, a) -> Nullable Function
-  function funcPath(path, x) {
-    return _funcPath(true, path, x);
+  //  functionImpl :: (String, String) -> Nullable Function
+  function functionImpl(type, name) {
```

Comment:
You'll notice this function is only used once. I kept it around for symmetry. However, once it's taken into use elsewhere, it'll have to include a check similarly to the one in `methodImpl` to avoid bogus input leading to an exception. Maybe I should just remove it?